### PR TITLE
feat: PennCNB json files also need to be checked if there are two items

### PR DESCRIPTION
**Pull request by kcho**
A function is added to check if there are two required items within the Penn CNB json file.

### CODE REVIEW

File: subject_files_status_for_dpdash.py [hunk 1]
```diff
@@ -34,6 +35,27 @@ def _latest_mtime(p: Path) -> str:
...
+
+    if len(data) != 2:
+        return False
+
+    session_batteries = [x['session_battery'] for x in data]
+    spllta_check = 'SPLLT-A' in session_batteries
+    nospllt_check = ('ProNET_NOSPLLT' in session_batteries or
+                     'PRESCIENT_NOSPLLT_C1' in session_batteries)
+
+    return spllta_check & nospllt_check
```
Comment (tashrifbillah) [inline subject_files_status_for_dpdash.py:56]: Why is it not `and`? I see you used `&`.

File: subject_files_status_for_dpdash.py [hunk 2]
```diff
@@ -34,6 +35,27 @@ def _latest_mtime(p: Path) -> str:
     return datetime.fromtimestamp(latest).strftime('%Y-%m-%d')
 
 
+def check_upenn_cnb(subject_raw_path: Path) -> bool:
+    '''Check if there are SPLLT-A and NOSPLLT in the Penn CNB json file'''
+    json_paths = list((subject_raw_path / 'surveys').glob('*UPENN.json'))
+
+    if len(json_paths) == 0:  # No file
+        return False
```
Comment (tashrifbillah) [inline subject_files_status_for_dpdash.py:43]: Here and below you return a `bool` but our previous `_is_file()` returned `1` or `0`:

https://github.com/AMP-SCZ/utility/blob/c55918b58bcf68a18a1ba024ccead04f62bd6bbb/subject_files_status_for_dpdash.py#L201-L203

Should you revise?

Comment (kcho) [reply subject_files_status_for_dpdash.py]: updated to return `int`

File: subject_files_status_for_dpdash.py (hunk 1 above)
Comment (kcho) [reply subject_files_status_for_dpdash.py]: `&` and `and` works the same. Replaced to `*` to return `int` in 3ca36d3. Thanks.

File: subject_files_status_for_dpdash.py [hunk 3]
```diff
@@ -35,25 +35,25 @@ def _latest_mtime(p: Path) -> str:
...
-        return False
+        return 0
 
     session_batteries = [x['session_battery'] for x in data]
     spllta_check = 'SPLLT-A' in session_batteries
     nospllt_check = ('ProNET_NOSPLLT' in session_batteries or
                      'PRESCIENT_NOSPLLT_C1' in session_batteries)
 
-    return spllta_check & nospllt_check
+    return spllta_check * nospllt_check
```
Comment (tashrifbillah) [inline subject_files_status_for_dpdash.py:56]: This is a very good design. I didn't know the `*` casts them to integer directly.

**Comment by tashrifbillah** [comment]
Thank you for the changes. Let's test it now. I want to deploy your PR locally before merging here. After I deploy, will you be able to confirm its functionality by looking at either the production or the mock DPdash? You will need to look at files' status dashboard activating the `date-avl-cnb-mriqc-kcho` configuration.

**Comment by tashrifbillah** [comment]
Hi, can I get a reply to my last comment?

**Comment by tashrifbillah** [comment]
Kevin will maintain `check_upenn_cnb()` function if errors arise in future. I am merging it.

**Pull request merged**